pkg/apis/marketplace/v1alpha1/operatorsource_types.go

```diff
@@ -39,7 +19,33 @@ type OperatorSourceSpec struct {
 	RegistryNamespace string `json:"registryNamespace,omitempty"`
 }
 
+// OperatorSourceStatus defines the observed state of OperatorSource
 type OperatorSourceStatus struct {
 	// Current phase of the OperatorSource object
 	CurrentPhase ObjectPhase `json:"currentPhase,omitempty"`
 }
+
```

Comment:
It can stay, but this file was 100% generated by the sdk. If we regenerate again, we will have to continue to manually modify it afterwards. Is that preferable?

Reply:
I may be misinterpreting your comment here. When we extend a type (add or remove new attributes), we manually edit the file and then run `operator-sdk generate k8s`.

Reply:
Good point. I take back my request.

Reply:
Yeah, @tkashem is right. This file gets generated by the SDK only when seeding the repository. Given that my request holds.

Reply:
Done